Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -787,6 +796,17 @@ public class LottieAnimationLayer: CALayer {
     }
   }
 
+  func loadAnimation(_ dotLottieAnimation: DotLottieFile.Animation) {
```

Comment:
Can this be private?
```suggestion
  private func loadAnimation(_ dotLottieAnimation: DotLottieFile.Animation) {
```